server/routers/users.py

```diff
@@ -0,0 +1,42 @@
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+from server.database.db_settings import SessionLocal
+from server.database.db_context import DBContext
+from server.database.schemas.users import UserCreate, UserGet
+from server.database.entities.users import User
+from server import main_repo
+
+router = APIRouter()
+
+
+def db_context():
+    db = SessionLocal()
+    try:
+        yield DBContext(db)
+    finally:
+        db.close()
+
+
+@router.post("/users/", response_model=UserGet)
```

Comment:
I think on the router we really need docs, including:
- Parameters
- Output
- Example of usage

Reply:
You mean the docs on the API requests?

Reply:
I've created an additional issue for it and probably you wouldn't do it